subsys/net/lib/lwm2m/Kconfig

```diff
@@ -253,6 +253,24 @@ endchoice
 
 endif # LWM2M_RESOURCE_DATA_CACHE_SUPPORT
 
+config LWM2M_SEND_SCHEDULER
```

Comment:
Can we add an overlay file to the LwM2M sample that enables this, and have it enabled in a configuration within `sample.yaml`? Currently it won't be built by anything so could bitrot easily.

Ideally we could have some test for this, although not sure how viable that would w/o a real broker. Perhaps `lwm2m_interop` could be extended for the purpose (which does run against real broker, though not in the on-PR CI).

Reply:
will take a look and see what I can do